challenge_spr/src/challenge_spr_states/bluff_game.py

```diff
@@ -19,53 +20,65 @@
 
 from riddle_game import hear, answer
 
+# def _turn_to_closest_entity(robot):
+#     # Reset the world model just to be sure
+#     robot.ed.reset()
+#
+#     operator = None
+#     while not operator:
+#         operator = robot.ed.get_closest_entity(radius=1.9, center_point=robot.base.get_location().extractVectorStamped())
+#         print operator
+#         if not operator:
+#             vth = 0.5
+#             th = 3.1415 / 10
+#             print "Turning %f radians with force drive" % th
+#             robot.base.force_drive(0, 0, vth, th / vth)
+#
+#     robot.base.force_drive(0, 0, 0, 0.5)
+#
+#     # Turn towards the operator
+#     current = robot.base.get_location()
+#     robot_th = current.frame.M.GetRPY()[2]  # Get the Yaw, rotation around Z
+#     desired_th = math.atan2(operator._pose.p.y() - current.frame.p.y(),
+#                             operator._pose.p.x() - current.frame.p.x())
+#
+#     # Calculate params
+#     th = desired_th - robot_th
+#     if th > 3.1415:
+#         th -= 2 * 3.1415
+#     if th < -3.1415:
+#         th += 2 * 3.1415
+#     vth = 0.5
+#
+#     # Turn
+#     robot.base.force_drive(0, 0, (th / abs(th)) * vth, abs(th) / vth)
 
 
-def _turn_to_closest_entity(robot):
-    # Reset the world model just to be sure
-    robot.ed.reset()
-
-    operator = None
-    while not operator:
-        operator = robot.ed.get_closest_entity(radius=1.9, center_point=robot.base.get_location().extractVectorStamped())
-        print operator
-        if not operator:
-            vth = 0.5
-            th = 3.1415 / 10
-            print "Turning %f radians with force drive" % th
-            robot.base.force_drive(0, 0, vth, th / vth)
+def turn_to_closest_entity(robot):
 
-    robot.base.force_drive(0, 0, 0, 0.5)
+    rospy.sleep(1.5)
```

Comment:
@SamAlexandrov why do we need this sleep? Can we make it configurable (or at least define it at the top of this file)